udata_hydra/routes/resources.py

```diff
@@ -29,7 +34,7 @@ async def get_resource_status(request: web.Request) -> web.Response:
     If resource is not found, respond with a 404 status code
     """
     try:
-        resource_id: str = request.match_info["resource_id"]
+        resource_id = str(uuid.UUID(request.match_info["resource_id"]))
     except Exception as e:
```

Comment:
I see you've reused the same `except Exception as e` above, I understand now ;) Code consistency!